app/api/helpers/mail.py

```diff
@@ -185,6 +185,7 @@ def send_email_session_state_change(email, session, mail_override: Dict[str, str
             mail = mail.copy()
             mail['subject'] = mail_override.get('subject') or mail['subject']
             mail['message'] = mail_override.get('message') or mail['message']
+            mail['bcc'] = mail_override.get('bcc') or mail['bcc']
```

Comment:
```suggestion
            mail['bcc'] = mail_override.get('bcc') or ]]
```

Reply:
sir do i need to remove this line or not.